aspnetcore/publishing/iis.md

```diff
@@ -94,21 +94,23 @@ Both `UseKestrel` and `UseIISIntegration` are required. Code calling *UseIISInte
 
 For more information on hosting, see [Hosting in ASP.NET Core](xref:fundamentals/hosting).
 
-### Setting IISOptions for the IISIntegration service
+### IIS options
 
-To configure *IISIntegration* service options, include a service configuration for *IISOptions* in *ConfigureServices*.
+To configure *IISIntegration* service options, include a service configuration for *IISOptions* in *ConfigureServices*:
 
 ```csharp
-services.Configure<IISOptions>(options => {
-  ...
+services.Configure<IISOptions>(options => 
+{
+    ...
 });
 ```
 
-| Option | Setting|
-| --- | --- | 
-| AutomaticAuthentication | If true, the authentication middleware will alter the request user arriving and respond to generic challenges. If false,the authentication middleware will only provide identity and respond to challenges when explicitly indicated by theAuthenticationScheme |
-| ForwardClientCertificate | If true and the `MS-ASPNETCORE-CLIENTCERT` request header is present, the `ITLSConnectionFeature` will be populated. |
-| ForwardWindowsAuthentication | If true, authentication middleware will attempt to authenticate using platform handler windows authentication. If false, authentication middleware won’t be added. |
+| Option                         | Default | Setting |
+| ------------------------------ | ------- | ------- |
+| `AutomaticAuthentication`      | `true`  | If `true`, the authentication middleware sets the `HttpContext.User` and responds to generic challenges. If `false`, the authentication middleware only provides an identity (`HttpContext.User`) and responds to challenges when explicitly requested by the `AuthenticationScheme`. Windows Authentication must be enabled in IIS for `AutomaticAuthentication` to function. |
+| `AuthenticationDisplayName`    | `null`  | Sets the display name shown to users on login pages. |
+| `ForwardClientCertificate`     | `true`  | If `true` and the `MS-ASPNETCORE-CLIENTCERT` request header is present, the `ITLSConnectionFeature` is populated. |
```

Comment:
Rather than ITLSConnectionFeature, saying HttpContext.Connection.ClientCertificate might be clearer. Few people use the features directly.